### DISCUSSION TITLE
Local install and marketplace empty

### DISCUSSION THREAD

**Discussion by sycured** [webstudio-is/webstudio-community#163] [General]
After a very long and crazy setup to have it running correctly, I ended with the marketplace (pages/sections) empty...
I don't know which option was missing but it's really painful to pass hours trying to set up for finally be impossible to have it fully working

Usage: working in transit/roaming, when no internet

**Comment by kof** [comment]
I think the real solution here is going to be the webstudio app that can work offline fully and sync back once internet is available

**Reply by juanivallejoss** [reply]
I'm building locally to use the premium features and learn on the go, but now, I can't download the sitemap "plugin".
Is there any way to export from webapp just the extension and "install" on local website?

or viceversa, 
export from local to online cloud, keep the premium features, install the extension, sync with local and keep working?

**Reply by kof** [reply]
What is a sitemap plugin? do you mean the marketplace project? you can currently only copy it in the builder in the cloud and paste it to your local project

**Reply by juanivallejoss** [reply]
Yes, the marketplace sitemap project. I'll give a try, I think it's just some code and adjustments to work locally too.
Thanks
